azblob/zc_sas_query_params.go

```diff
@@ -190,6 +195,18 @@ func newSASQueryParameters(values url.Values, deleteSASParametersFromValues bool
 			p.contentLanguage = val
 		case "rsct":
 			p.contentType = val
+		case "skoid":
+			p.userDelegationKey.SignedOid = val
```

Comment:
these are SAS query parameters, no need for the intermediate `userDelegationKey`

Reply:
Originally, I had implemented this with the parameters rather than the intermediate userDelegationKey but found that the latter felt like a cleaner approach. I'm happy to revert, however.